curriculum/challenges/english/25-front-end-development/quiz-dictionaries-and-sets/67f412ac59601c4151b763da.md

```diff
@@ -17,439 +17,498 @@ To pass the quiz, you must correctly answer at least 18 of the 20 questions belo
 
 #### --text--
 
-Placeholder question
+What is a dictionary?
 
 #### --distractors--
 
-Placeholder distractor 1
+A data structure that only holds strings and lists.
 
 ---
 
-Placeholder distractor 2
+A data structure that is a collection of key-value pairs that only accept large numbers.
 
 ---
 
-Placeholder distractor 3
+A data structure that only holds a collection of tuples.
 
 #### --answer--
 
-Placeholder answer
+A data structure that is a collection of key-value pairs.
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which of the following is the correct syntax for a dictionary?
 
 #### --distractors--
 
-Placeholder distractor 1
+```py
+dictionary = {
+    <key1: value1>,
+    <key2: value2>'
+}
+```
 
 ---
 
-Placeholder distractor 2
+```py
+dictionary = {
+    (key1: value1),
+    (key2: value2)'
+}
+```
 
 ---
 
-Placeholder distractor 3
+```py
+dictionary = {
+    [key1: value1],
+    [key2: value2]'
+}
+```
 
 #### --answer--
 
-Placeholder answer
+```py
+dictionary = {
+    'key1': 'value1',
+    'key2': 'value2'
```

Comment:
```suggestion
    key1: value1,
    key2: value2
```
I think it makes sense not to use quotes for consistency, because of how the distractors are written and because keys can be non-strings. Also, for one of the following questions, using quotes the right answer would be `dictionary['key']` instead of the current `dictionary[key]`.

Reply:
@zairahira 

The decision to remove the quotes was here 👍🏾